pyccel/symbolic/lambdify.py

```diff
@@ -66,7 +66,7 @@ def cse(expr):
             if not all(size):
                 raise ValueError('Unable to find range of index')
             name = str(vars_new[i].base)
-            var = Symbol(name)
+            var = PyccelSymbol(name)
```

Comment:
Cannot we merge these two lines into a single one, where we do not call `str()`?

```python
var = PyccelSymbol(vars_new[i].base)
```

Reply:
5f5d919